src/CalcViewModel/Common/TraceLogger.cpp

```diff
@@ -152,7 +149,7 @@ namespace CalculatorApp
         LogLevel2Event(EVENT_NAME_WINDOW_ON_CREATED, fields);
     }
 
-    void TraceLogger::LogModeChange(ViewMode mode) const
+    void TraceLogger::LogModeChange(ViewMode mode)
     {
         if (!GetTraceLoggingProviderEnabled())
```

Comment:
Not your code, but--checking whether the provider is enabled is not necessary. LoggingChannel does this check internally.

Reply:
I will modify that in this PR